client/landing/stepper/declarative-flow/internals/steps-repository/importer-wordpress/index.tsx

```diff
@@ -1,12 +1,28 @@
 import WordpressImporter from 'calypso/blocks/importer/wordpress';
 import { Step } from 'calypso/landing/stepper/declarative-flow/internals/types';
+import { MigrationAssistanceModal } from '../../components/migration-assistance-modal';
 import { withImporterWrapper } from '../importer';
 import './style.scss';
 
 const Importer = withImporterWrapper( WordpressImporter );
 
 const ImporterWordpress: Step = function ( props ) {
-	return <Importer importer="wordpress" { ...props } />;
+	const queryParams = new URLSearchParams( props?.data?.path || window.location.search );
```

Comment:
Can you investigate if we can use [useQuery](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/hooks/use-query.ts)?

```
const queryParams = useQuery();
const migrateFrom = queryParams.get( 'from' );
```